rules/network/command_and_control_cobalt_strike_beacon.toml

```
@@ -0,0 +1,56 @@
+[metadata]
+creation_date = "2020/07/06"
+ecs_version = ["1.5.0"]
+maturity = "production"
+updated_date = "2020/07/06"
+
+[rule]
+author = ["Elastic"]
+description = """
+Cobalt Strike is a threat emulation platform commonly modified and used by adversaries to conduct network attack and
+exploitation campaigns. This rule detects a network activity algorithm leveraged by Cobalt Strike implant beacons for
+command and control.
+"""
+false_positives = [
+    """
+    This rule should be tailored to either exclude systems, as sources or destinations, in which this behavior is
+    expected.
+    """,
+]
+index = ["packetbeat-*"]
+language = "lucene"
+license = "Elastic License"
+name = "Cobalt Strike Command and Control Beacon"
+note = "This activity has been observed in FIN7 campaigns."
+references = [
+    "https://blog.morphisec.com/fin7-attacks-restaurant-industry",
+    "https://www.fireeye.com/blog/threat-research/2017/04/fin7-phishing-lnk.html",
+]
+risk_score = 73
+rule_id = "cf53f532-9cc9-445a-9ae7-fced307ec53c"
+severity = "high"
+tags = ["Elastic", "Network"]
+type = "query"
+
+query = '''
+event.category:(network OR network_traffic) AND type:(tls OR http) AND network.transport:tcp AND destination.domain : /[a-z]{3}.stage.[0-9]{8}\.(.*)/
```

```suggestion
event.category:(network OR network_traffic) AND type:(tls OR http) AND network.transport:tcp AND destination.domain : /[a-z]{3}.stage.[0-9]{8}\..*/
```

Are the parens needed here?

---

I added the `()` because in the past, Kibana had been a bit greedy with some of the regex and the parens restricted better.

That said, I just re-ran the test and got the same result...so maybe not for this situation. LMK if the standard is no parens and I'll remove 👍

---

Yea, using parens would make sense if we needed to address a group (or sub-group) pattern, but in this case, I think we are safe to remove them.